pkgs/development/compilers/rust/rustc.nix

```diff
@@ -52,6 +42,9 @@ stdenv.mkDerivation {
   # versions.
   RUSTC_BOOTSTRAP = "1";
 
+  # Increase codegen units to introduce parallelism within the compiler.
+  RUSTFLAGS = "-Ccodegen-units=10";
```

Comment:
Maybe we should set this in `preBuild`? Spares us the re-exporting in `checkPhase`